frontend/src/App.js

```diff
@@ -54,6 +61,21 @@ function App() {
             </>
           )
         }
+        {
+          hasRole(currentUser, "ROLE_USER") && (
+            <>
+              <Route exact path="/hotels/" element={<HotelIndexPage />} />
```

Comment:
look at the line above where ucsbdates is used, I think you're forgetting the list after /